app/views/user_mailer/delete_work_notification.text.erb

```diff
@@ -1,9 +1,12 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
-  
-Your work "<%= @work.title.html_safe %>" was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? "the request of " + text_pseud(User.current_user.default_pseud) : "your request" %>.
-  
-If you have questions, please contact Support at <%= root_url %>support.
-  
-Attached is a copy of your work for your reference.
-<% end %>
\ No newline at end of file
+<%= t('.hello', user:  @user.default_pseud.byline) %>
+<%= (@work.pseuds.count > 1 && @user != User.current_user) ?
+t('.part1_other',
+        title: @work.title,
+        pseud: User.current_user.default_pseud) :
+t('.part1_yourself',
+        title: @work.title) %>
+<%= t('.part2', support: t('.support')+' '+new_feedback_report_url) %>
```

Comment:
Oh, good point about the translation. I have a really hard time imagining a scenario in which that would be the case, since the entire paragraph is dedicated to telling the user to contact us, but better safe than sorry.

I've been letting the single quotes thing go on these because they're so old -- as you can see, they started off as double quotes and then got switched to single, when that's what we were planning to do. If james is feeling like updating all nine of these, that would be great, but otherwise we can just do them all in one fell swoop at the end instead of tying them up even longer.